src/graphql/schema.js

```diff
@@ -46,12 +47,338 @@ const ProficiencyTC = composeMongoose(Proficiency);
 const RaceTC = composeMongoose(Race);
 const RuleTC = composeMongoose(Rule);
 const RuleSectionTC = composeMongoose(RuleSection);
+const SkillTC = composeMongoose(Skill);
 const SpellTC = composeMongoose(Spell);
 const SubclassTC = composeMongoose(Subclass);
 const SubraceTC = composeMongoose(Subrace);
 const TraitTC = composeMongoose(Trait);
 const WeaponPropertyTC = composeMongoose(WeaponProperty);
 
+// TODO: Figure out how to use commented out relations without breaking GraphQL Playground.
+// Commented out relations lead to circular dependencies. This causes graphql introspection to enter an infinite recursive loop,
+// breaking GraphQL Playground.
+
+AbilityScoreTC.addRelation('skills', {
+  resolver: () => SkillTC.mongooseResolvers.findMany(customizationOptions),
+  prepareArgs: {
+    filter: source => ({
+      _operators: {
+        index: { in: source.skills.map(skill => skill.index) },
+      },
+    }),
+  },
+  projection: { skills: true },
+});
+
+BackgroundTC.addRelation('starting_proficiencies', {
+  resolver: () => ProficiencyTC.mongooseResolvers.findMany(customizationOptions),
+  prepareArgs: {
+    filter: source => ({
+      _operators: {
+        index: { in: source.starting_proficiencies.map(prof => prof.index) },
+      },
+    }),
+  },
+  projection: { starting_proficiencies: true },
+});
+
+ClassTC.addRelation('class_levels', {
+  resolver: () => LevelTC.mongooseResolvers.findMany(customizationOptions),
+  prepareArgs: {
+    filter: source => ({
+      class: {
+        index: source.index,
+      },
+    }),
+  },
+  projection: { index: true },
+});
+
+ClassTC.addRelation('spells', {
+  resolver: () => SpellTC.mongooseResolvers.findMany(customizationOptions),
+  prepareArgs: {
+    filter: source => ({
+      classes: [{ index: source.index }],
+    }),
+  },
+  projection: { index: true },
+});
+
+ClassTC.addRelation('proficiencies', {
+  resolver: () => ProficiencyTC.mongooseResolvers.findMany(customizationOptions),
+  prepareArgs: {
+    filter: source => ({
+      classes: [{ index: source.index }],
+    }),
+  },
+  projection: { index: true },
+});
+
+// EquipmentTC.addRelation('equipment_category', {
+//   resolver: () => EquipmentCategoryTC.mongooseResolvers.findOne(customizationOptions),
+//   prepareArgs : {
+//     filter: source => ({
+//       index: source.equipment_category.index
+//     })
+//   },
+//   projection: { equipment_category: true }
+// });
+
+// EquipmentTC.addRelation('gear_category', {
+//   resolver: () => EquipmentCategoryTC.mongooseResolvers.findOne(customizationOptions),
+//   prepareArgs : {
+//     filter: source => ({
+//       index: source.gear_category.index
+//     })
+//   },
+//   projection: { gear_category: true }
+// });
```

Comment:
I feel like there is an open proposal in the DB to consolidate these. Not sure if that helps.